actioncable/lib/action_cable/helpers/action_cable_helper.rb

```diff
@@ -9,7 +9,7 @@ module ActionCableHelper
       #     <%= javascript_include_tag 'application', 'data-turbolinks-track' => true %>
       #   </head>
       #
-      # This is then used by Action Cable to determine the url of your WebSocket server.
+      # This is then used by ActionCable to determine the url of your websocket server.
```

Comment:
I have no idea how this happened... something funky must have happened while I was rebasing...